bridgetown-website/netlify.toml

```diff
@@ -0,0 +1,40 @@
+[build]
```

Comment:
Weirdly if you have a different base, you need two `netlify.tml` files -.- Took me so long to figure out why the headers weren't being picked up on.